src/scripts/Controller.js

```diff
@@ -608,6 +640,9 @@ export default class SitemapController {
 
 	importSitemap() {
 		// cancel submit if invalid form
+		let validator = this.getFormValidator();
```

Comment:
I think we should have this revalidation in readBlob but not here. So we have actual state in form.